crt_portal/static/sass/custom/form.scss

```diff
@@ -10,6 +10,8 @@ form#filters-form {
     width: 100%; // IE11 fix
     margin: 0;
   }
+}
```

Comment:
There are a lot of CSS changes in this file which makes me uncomfortable. Looks like the vast majority (if not everything) are whitespace changes -- can you please check your CSS linter? It looks like some of the indentations are incorrectly collapsed here.